assets/shell.css

```
@@ -235,6 +243,57 @@
   .bfp-content { padding: 70px 18px 120px; }
   .bfp-brand { font-size: 20px; }
   .bfp-brand-lens { font-size: 17px; }
+
+  /* Inline code is normally kept on one line for readability. On narrow
+     screens, long header names, URLs, and commands can otherwise widen the
+     entire document and make the article appear shifted off-screen. */
+  .bfp-content code:not(pre code) {
+    white-space: normal;
+    overflow-wrap: anywhere;
+    word-break: break-word;
```

Stylelint (17.14.0)

[error] Deprecated keyword "break-word" for property "word-break" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)